packages/actions/routes/web.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+use Filament\Actions\Imports\Http\Controllers\DownloadImportFailureCsv;
+use Illuminate\Support\Facades\Route;
+
+Route::get('/filament/imports/{import}/failed-rows/download', DownloadImportFailureCsv::class)
+    ->name('filament.imports.failed-rows.download')
+    ->middleware(['web', 'auth']);
```

Comment:
shouldn't this use current panel auth? not just auth?

Reply:
Nope, this is not a panels feature, this is part of filament/actions. We don't have access to the panel middleware